### PR TITLE
[W4.11e][F11-1]Zhang Yezhong

### PR DESCRIPTION
**Pull request by YipZong**
ready for review

### CODE REVIEW

**Comment by YipZong** [comment]
ready for review

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +43,42 @@ private void assertAreUnique(Object... objects) {
...
+
+        // one element( has null)
+        assertHasNull((Object) null); //one null object
+
+        // one element (no null)
+        assertDoNotHaveNull(1);
+        assertDoNotHaveNull("a");
+        assertDoNotHaveNull("");
+
+        // 2 elements one null object + one number
```
Comment (liliwei25) [inline test/java/seedu/addressbook/common/UtilsTest.java:60]: These comment will be confusing for others when you have assertDoNotHaveNull together with assertHasNull below

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -43,4 +43,42 @@ private void assertAreUnique(Object... objects) {
...
+        assertDoNotHaveNull("a", 1);
+        assertDoNotHaveNull("", 1);
+        assertHasNull((Object) null, 1);
+        assertHasNull((Object) null, "a");
+        assertHasNull((Object) null, "");
+
+        // more elements
```
Comment (liliwei25) [inline test/java/seedu/addressbook/common/UtilsTest.java:67]: Good job having test cases for more than one or two objects 👍

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -43,4 +43,42 @@ private void assertAreUnique(Object... objects) {
...
+        assertHasNull(1, null, "a");
+        assertHasNull(222, null, "a");
+        assertHasNull(null, "", "aaa", 333);
+    }
+
+    private void assertHasNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
+    private void assertDoNotHaveNull(Object... objects) {
```
Comment (liliwei25) [inline test/java/seedu/addressbook/common/UtilsTest.java:79]: Maybe can go with assertNoNull or something shorter?

**Review by liliwei25 — COMMENTED** [review]
👍 

Please read the comments and close the PR :)

File: test/java/seedu/addressbook/common/UtilsTest.java (hunk 1 above)
Comment (YipZong) [reply test/java/seedu/addressbook/common/UtilsTest.java:60]: Noted. Thank you!

File: test/java/seedu/addressbook/common/UtilsTest.java (hunk 2 above)
Comment (YipZong) [reply test/java/seedu/addressbook/common/UtilsTest.java:67]: Thank you!

File: test/java/seedu/addressbook/common/UtilsTest.java (hunk 3 above)
Comment (YipZong) [reply test/java/seedu/addressbook/common/UtilsTest.java:79]: Yea good idea!

**Pull request closed without merging**